apps/src/templates/instructions/InlineAudio.jsx

```diff
@@ -87,6 +87,8 @@ const InlineAudio = React.createClass({
     });
 
     audio.addEventListener("error", e => {
+      // e is an instance of a MediaError object
+      trackEvent('InlineAudio', 'error', e.target.error.code);
```

Comment:
It is admittedly fairly minor feedback, but it's also all the information we are given.

We should be able to see in GA what play event this error is associated with so we'll have the src url. I'm not sure what else we can add.